tailscale.go

```diff
@@ -349,7 +349,7 @@ func TsnetSetEphemeral(sd C.int, e int) C.int {
 }
 
 //export TsnetSetLogFD
-func TsnetSetLogFD(sd C.int, fd int) C.int {
+func TsnetSetLogFD(sd C.int, fd C.int) C.int {
```

Comment:
Maybe:
```suggestion
func TsnetSetLogFD(sd, fd C.int) C.int {
```

now that they're the same.